cs464-group-project/src/pages/HomeComps/StadiumCap.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+
+import { Bar } from 'react-chartjs-2';
+import { Chart as ChartJS } from 'chart.js/auto';
+import { Chart } from 'react-chartjs-2';
+
+const backgroundColors = [
+  'rgba(75, 192, 192, 0.8)',
+  'rgba(255, 99, 132, 0.8)',
+  'rgba(54, 162, 235, 0.8)',
+  'rgba(255, 206, 86, 0.8)',
+  'rgba(153, 102, 255, 0.8)',
+  'rgba(255, 159, 64, 0.8)',
+  'rgba(128, 128, 128, 0.8)',
+  'rgba(70, 130, 180, 0.8)',
+  'rgba(34, 139, 34, 0.8)',
+  'rgba(255, 140, 0, 0.8)',
+  'rgba(138, 43, 226, 0.8)',
+];
+
+const borderColors = [
+  'rgba(75, 192, 192, 1)',
+  'rgba(255, 99, 132, 1)',
+  'rgba(54, 162, 235, 1)',
+  'rgba(255, 206, 86, 1)',
+  'rgba(153, 102, 255, 1)',
+  'rgba(255, 159, 64, 1)',
+  'rgba(128, 128, 128, 1)',
+  'rgba(70, 130, 180, 1)',
+  'rgba(34, 139, 34, 1)',
+  'rgba(255, 140, 0, 1)',
+  'rgba(138, 43, 226, 1)',
+];
+
+function StadiumCap({ prop }) {
+  const [displayedTeams, setDisplayedTeams] = useState([]);
+
+  //https://stackoverflow.com/questions/31162606/how-to-detect-screen-size-for-responsive-web-design
+  useEffect(() => {
```

Comment:
If you use bootstap grip/cols/and row, you wont need this when resizing window

Reply:
I wanted to display fewer teams as the screen size changes.  So what the code basically does is when the screen is being resized it will call the function to get only top 10 or top 5.  I can probably hide the other teams using css but my thought process was that it might be easier to just cut the array and display it again. 
![image](https://github.com/CS464-Group-Project/CS464_GroupProject/assets/122697354/6a920459-3c74-46a7-a26a-7b02cd559ff5)